dot/network/host_test.go

```diff
@@ -273,24 +274,32 @@ func TestExistingStream(t *testing.T) {
 	require.NoError(t, err)
 
 	time.Sleep(TestMessageTimeout)
-	require.NotNil(t, handlerB.messages[nodeA.host.id()], "node B timeout waiting for message from node A")
+
+	messages, _ := handlerB.messagesFrom(nodeA.host.id())
+	assert.Len(t, messages, 1)
+	require.NotNil(t, messages, "node B timeout waiting for message from node A")
```

Comment:
Should you instead check for the `ok` returned by `handlerB.messagesFrom(nodeA.host.id())`??